include/pisa/query/term_processor.hpp

```diff
@@ -16,6 +18,30 @@ namespace pisa {
 
 using term_id_type = uint32_t;
 
+std::function<std::string(std::string)> get_term_processor(std::optional<std::string> const& type)
```

Comment:
Some versions of GCC are complaining about multiple definitions of this, could you check Travis?

Reply:
Will fix